docs/EMERGENCY.md

```diff
@@ -0,0 +1,17 @@
+# Emergency mode
+
+ES-Operator will not operate a non-green cluster. Therefore, there are certain situations that require human intervention to recover an Elasticsearch cluster running with ES-Operator.
+
+## Stopping the bleeding
+
+The first and foremost goal should be to get your cluster back into green state. The [cluster reroute API](https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-reroute.html#cluster-reroute-api-request-body) can be used to temporarily allocate an empty primary, until you have recovered your data.
+
+If, for whatever reason, this is not feasible, you will need to switch from auto-pilot to manual steering. For this, please do the following:
+
+## Manual Recovery Steps
+
+1. Stop the ES-Operator.
+2. If necessary, change the replicas in the stateful set attached to your EDS (it has the same name as the EDS) - **WARNING! Once the ES-Operator is started again, it will rigorously overwrite this setting, potentially leading to data loss by terminating pods without draining!**
+3. After manual steering, in your EDS, disable auto-scaling (`spec.scaling.enabled: false`) and set `spec.replicas` to the current number of replicas in the STS. 
```

Comment:
@otrosien just to confirm. First change the STS settings (e.g. scaling) and then disable the auto-scaling in the EDS and change the replica count there?

What's the impact of leaving scaling enabled for EDS if the ES-Operator is off?

Aren't we missing something between steps 3 and 4? e.g. getting the cluster back to green first and then re-enabling ES-Operator?

Reply:
that is what I meant with "After manual steering" - as mentioned above, human intervention is needed to get cluster state back to green. I can extend this a bit more.

Reply:
> What's the impact of leaving scaling enabled for EDS if the ES-Operator is off?

If you re-enable the ES Operator with auto-scaling on, it will assume the replica in its scaling "status" is correct and overwriting the STS relica setting. This is most likely incorrect, therefore switching to manual scaling, and setting the EDS value to what you've manually scaled the STS to.

Reply:
@otrosien I'd be more explicit. Otherwise it might be unclear that there are some implicit operations and time gap between point 3 and 4.

From your answer I understand that the problem is not having auto-scaling on while the ES-Operator is off, rather that it would be a problem starting ES-Operator with Auto-scaling on. 

I found it interesting that the first thing you do after shutting down ES-Operator is to disable auto-scaling as if it were a critical step, but it seems to only be critical before restarting ES-Operator.

So perhaps 1 and 2 are to be done before the cluster recovery and 3, 4 and 5 after the cluster recovery? It's important to be clear when the cluster recovery actions should be taken.